tools/qontract_cli.py

```diff
@@ -491,6 +491,57 @@ def service_owners_for_rds_instance(ctx, aws_account, identifier):
     print_output(ctx.obj['output'], service_owners, columns)
 
 
+@get.command()
+@click.pass_context
+def sre_checkpoints(ctx):
+    sre_checkpoints_raw = queries.get_sre_checkpoints()
+    apps = queries.get_apps()
+
+    def full_name(app):
```

Comment:
Not a big deal here, but internal functions are very hard to test. We can accept it here.